templates/home.html

```diff
@@ -70,7 +70,13 @@
       <li>
         <a href='{{ url_for_user(user) }}'>
           <figure>
-            <img src={{ user.avatar }} width=100 height=100 onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'" alt="">
+            <img 
+              src={{ user.avatar }}
+              title={{ user.avatar }}
+              width=100
+              height=100
+              onerror="this.src='{{ url_for('static', filename='images/avatar.png') }}'"
+              alt>
```

Comment:
Shouldn't we insert some alternative text here for accessibility purpose? Like "photo of {{ user }}"

Reply:
true and done :)